app/regen/cmd/genaccounts.go

```diff
@@ -19,6 +17,7 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/spf13/cobra"
```

Comment:
gofmt needed

Reply:
It is up-to-date with gofmt format, I cross-checked too